assets/queries/ansible/aws/IAM_policy_all/query.rego

```diff
@@ -0,0 +1,27 @@
+package Cx
+
+CxPolicy [  result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  awsApiGateway := task["community.aws.iam_managed_policy"]
+  contains(awsApiGateway.state, "present")
+  resource := awsApiGateway.policy.Statement[_].Resource
+  contains(resource, "*")
```

Comment:
Besides the policy affecting all Resources, the query checks if the effect is to allow some action in them. You need to check if the Effect is Allow as well

Reply:
Once again, you are right. However in terraform query that validation was not included. I'll change both